docs/modules/security/pages/native-client-security.adoc

```diff
@@ -116,6 +116,13 @@ the client endpoint address (`ClusterEndpointPrincipal`), and client roles (`Clu
 The default permission policy allows to use comma separated names in the `principal`
 attribute configuration.
 
+[NOTE]
+====
+When adding names to the `principal` attribute, do not include spaces before or after the name.
+
+Hazelcast cannot currently map names that start or end with a space, or that include commas.
```

Comment:
This needs to be corrected. Names with spaces are correctly handled. You can even have names starting and ending with whitespace (e.g. `" admin "`).

What is not supported is automatic space removal.
e.g. configuring permission with `principal=" admin ,devel"` means that the following role names will be granted the permission: `" admin "`, `"devel"`. The `"admin"` role will not be granted the permission.

I.e. the spaces in the `principal` configuration are significant.

Reply:
Thanks, @kwart I have updated the file.